### ISSUE TITLE
Formatting results: Dates

### ISSUE THREAD

**Issue by tecosaur** [nico202/ob-julia#5]
The string output of `Date` and `DateTime` objects is fine at the moment.

```org
#+begin_src julia
DateTime(2013,7,1,12,30,59)
#+end_src

#+RESULTS:
: 2013-07-01T12:30:59

#+begin_src julia
Date(2013,7,1)
#+end_src

#+RESULTS:
: 2013-07-01
```

But I wonder if it would be nice to use Org timestamps here? If so maybe we could do something like this:

```julia
julia> using Dates

julia> Dates.format(Date("2015-01-01"), "[yyyy-mm-dd e]")
"[2015-01-01 Thu]"

julia> Dates.format(DateTime("2015-01-01T01:22"), "[yyyy-mm-dd e HH:MM]")
"[2015-01-01 Thu 01:22]"
```

**Comment by nico202** [comment]
Fixed in 6948df7

**Comment by tecosaur** [comment]
Nice!

FYI if you say something like `Closes #5` in the commit message (can be in part of the description) GitHub will automatically close that issue and link the commit :slightly_smiling_face:.
